web/modules/custom/usa_contact_center_api/src/Controller/USAContactCenterController.php

```diff
@@ -55,6 +55,47 @@ public function getDeletedNodes() {
       $query->condition('deleted_time', $date, '>=');
     }
     $result = $query->execute()->fetchAll();
+
+    // We want to show a property of "updated_time" instead of "deleted_time" as per USAGOV-1936.
+    foreach ( $result as &$item ) {
+      $item = (array)$item;
+      $item['updated_time'] = $item['deleted_time'];
```

Comment:
My note was unclear. This should stay as-is (provide 'deleted_time' in the API).

Reply:
Ah, I understand now - this is now fixed